exchangelib/autodiscover.py

```diff
@@ -429,7 +429,7 @@ def _parse_response(response, encoding='utf-8'):
     # There are three possible protocol types: EXCH, EXPR and WEB. EXPR is for EWS. See
     # http://blogs.technet.com/b/exchange/archive/2008/09/26/3406344.aspx
     for protocol in protocols:
-        if get_xml_attr(protocol, '{%s}Type' % RESPONSE_NS) != 'EXPR':
+        if get_xml_attr(protocol, '{%s}Type' % RESPONSE_NS) not in ('EXCH', 'EXPR'):
```

Comment:
Ok, maybe I have configured it in a wrong way. I am using Exchange 2010 in a local test environment with internal URL's. Outlook Anywhere is disabled, that's the reason auto discover couldn't find EXPR. You can close this PR if you feel there is no need to address cases where Outlook Anywhere is disabled or client communication over RPC protocol.I agree most of the users won't configure Exchange Server this way. Thank you very much for looking into this. Appreciate it!